### DISCUSSION TITLE
why are entities names in german on Buderus?

### DISCUSSION THREAD

**Discussion by g6094199** [emsesp/EMS-ESP32#1269] [General]
Hi,

i have a buderus and a junkers intallation. On the Junkers the entities have english names and on the Buderus they are in german. 

![grafik](https://github.com/emsesp/EMS-ESP32/assets/37833326/6d57d261-66b2-4b03-811b-8862c77ff1b1)

it doesnt matter what language the webgui is set to. 
i dont get is.

any hints whats up?

sascha

**Comment by g6094199** [comment]
dont mind. found the setting.

![grafik](https://github.com/emsesp/EMS-ESP32/assets/37833326/e413e692-b49d-4e5e-bbe2-f135f7846777)


maybe it would make sense to set the entity naming to the set gui language?!

**Reply by proddy** [reply]
This is by design. The web language is the UI only, i.e. the HTML. The entities come from the core C++ code and are also used in MQTT. API etc. Changing the language from the web is harmless as it's just for visibility and non-volatile. Change the entity language in the Settings page changes the names in MQTT and API so can break any previous configurations when switched. 

What do you mean Junkers is English and Buderus is German? Can you show an example?
